templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete car</h1>
+
+  <p>Are you sure you want to delete the car: {{ car.model }}?</p>
+  <p><i>All information about this car will be deleted!</i></p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger"/>
```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on.